### ISSUE TITLE
changed policies are not propagated to the nsms

### ISSUE THREAD

**Issue by domsj** [openvstorage/alba#307]
Because the nsm doesn't know this new policy it blocks writes that may violate the policy (as a defency mechanism).

This resulted in the maintenance process not being able to do its work:

```
2016-08-05 09:34:45 6548 +0200 - perf-roub-04 - 20590/0 - alba/maintenance - 614580 - info - Exn while repairing object (~namespace_id:14178 ~object ~name:"r \000\000\000Z\142qU\004q\161\238\158h\154Iy\205
g{\152\224\209\1596\208\136\220w4\004G\159Mv\001\001\000\000\000\001\000\000\000\000\000\000\000" ~object_id:"\001y\240\254H\193L\208*\151\215\nPg'\150C3\022\173\018\144G\211\154\222\223r\213\216\219\018")
, will now try object rewrite: Nsm_model.Err.Nsm_exn(14, "Attempting to use 4 disks of node 5Yo6bZqwcZywSQZlXE3aU6pZLfAWLTC5 while max 3 permitted ([(Some 17l); (Some 7l); (Some 25l); (Some 14l); (Some 1l)
; (Some 24l);\n (Some 12l); (Some 20l); (Some 2l); (Some 18l); (Some 27l); (Some 4l)])"); backtrace:; Raised at file "queue.ml", line 86, characters 29-34; Called from file "src/tools/lwt_pool2.ml", line 9
8, characters 25-46
```

It could probably also result in writes from the proxy failing due to this.

Workaround: don't change your policy, and if you did change it back to the original value.

**Comment by domsj** [comment]
The issue is actually even worse: once an object has been written with a certain amount of disks per node it can never get a higher amount of disks per node, regardless of what is in the policy.
The policies ain't even propagated to the nsm when it's created...

**Comment by domsj** [comment]
reducing priority to normal because #309 has been merged

**Comment by wimpers** [comment]
After discussion with @domsj  and @toolslive  moved to RC2

**Comment by domsj** [comment]
fixed by #486
